test/client/before-test.js

```diff
@@ -138,6 +138,31 @@
         ok(passed);
     };
 
+    window.createIframeForTest = function (src, elLocation) {
```

Comment:
In the [test/client/fixtures/utils/dom-test.js](https://github.com/DevExpress/testcafe-hammerhead/pull/1255/commits/07311e417c4e158f3e986b2d29a05a45db5d7b7c#diff-b6a1dd5e49393e1752632fcf8747eb8c) file the createIframeForTest function is used 21 times.
including for nested iframes - https://github.com/DevExpress/testcafe-hammerhead/pull/1255/commits/07311e417c4e158f3e986b2d29a05a45db5d7b7c#diff-b6a1dd5e49393e1752632fcf8747eb8cR682